ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -44,6 +55,12 @@ public void clickButton(String buttonTitle) {
 		buttonElement.click();
 	}
 
+	public void clickModalDialogButton(String buttonTitle) {
```

Comment:
This method is redundant when you have `getModalDialogButton` on which you can invoke `click()` or whatever action needed.

Reply:
I agree. I have left it as it is because it is about overall concept of SyndesisPageObject class. There are also other click methods. If we change this concept then also this method will be changed.

Reply:
I'm not sure if this approach is completely in line with overall concept of SyndesisPageObject class. In this context you should rather instantiate ModalDialogPage page object, get button on it and then click it. Which leads me to the question if getModalDialogButton and clickModalDialogButton methods aren't both redundant.

Reply:
We'll solve this in another PR. Maybe removing all ...click() methods from SyndesisPageObject would be the best.

Reply:
OK